docs/team/EthanWong2212.md

```diff
@@ -0,0 +1,38 @@
+# Wong Heng Chin - Project Portfolio Page
+
+## Overview
+#### Project: Nav@NUS
+Nav@NUS is an application which helps users navigate themselves around NUS through the NUS shuttle services. The application runs on the Command Line Interface (CLI).
+
+## Summary of Contributions
+Given below are my contributions to the project.
+
+#### New Feature: Show buses available at bus stop
+ * What it does: Allows users to find the buses available at a specific bus stop.
+ * Justification: This is an important feature as it allows users easy access to the buses available at the stop in the event they are planning to change buses.
+ 
+#### New Feature: Execution of favourite command
+ * What it does: Allows user to execute commands directly from their list of favourite commands.
+ * Justification: This is an important feature as the command to execute a favourite command directly from the list can be significantly shorter. This helps users obtain required information quicker and reduces the posibility of spelling errors when typing long conventional commands.
+
+
+#### Code Contributed
+* This [RepoSense link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=ethanwong2212&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other) 
+reflects my code contribution.
+
+#### Enhancements implemented   
+
+#### Contributions to documentation
+* Added documentation for features `/route`and `/descfav`.
+
+#### Contributions to the DG
+
+
```

Comment:
You might want to either remove the empty sections or add something to it.

Reply:
things will be added in future